community-website/src/community-project-boilerplate-docgen/src/getting-started.md

```diff
@@ -28,7 +28,7 @@ It contains sample data for an e-commerce website.
 This guide also expects you to have a working angular application. You can use the official [angular CLI](http://cli.angular.io) to bootstrap a new application this way:
 
 ```sh
-npm install -g @angular/cli
+npm install -g @angular/cli@6
```

Comment:
Do we really need the `xx@6`? Since the latest version available on NPM is the `6.0.3`.

Reply:
Yes so that when 7 is out on the latest tag, this guide still works!